src/pages/signup.tsx

```diff
@@ -42,11 +44,47 @@ export default function Signup() {
 
   const [serverErrorState, setServerError] = useState<string | null>(null);
 
-  const onSubmitHandler: SubmitHandler<FormValues> = async () => {
+  const router = useRouter();
+
+  const onSubmitHandler: SubmitHandler<FormValues> = async (formData) => {
+    const formDataJson = JSON.stringify(formData);
+
     try {
       // @TODO Implement submit
       // await onSubmit(formData.email, formData.password);
 
+      // Local Test
+      // axios({
+      //   method: 'post',
+      //   url: '/api/v1/auth/signup',
+      //   baseURL: 'http://localhost:3000',
+      //   data: {
+      //     email: formData.email,
+      //     password: formData.password,
+      //     fullName: formData.fullName,
+      //   },
+      // })
+      //   .then((response) => {
+      //     console.log(response);
+      //   })
+      //   .catch((error) => {
+      //     console.log(error);
+      //   });
+
+      axios
+        .post('/api/v1/auth/signup', {
+          email: formData.email,
+          password: formData.password,
+          fullName: formData.fullName,
+        })
+        .then(function (response) {
+          console.log(response);
+        })
+        .catch(function (error) {
+          console.log(error);
+        });
+
+      // router.push('/test');
```

Comment:
Same story as in the `login` page, remove all unused code comments, and refactor the code block to use `async` and `await` instead of `then` and `catch` and take advantage of the `try {} catch {}` block